OrbitCore/Message.h

```diff
@@ -40,6 +40,9 @@ enum MessageType : int16_t
     Msg_NumFlushedItems,
     Msg_NumInstalledHooks,
     Msg_Callstack,
+    // TODO: this should basically be the same.
```

Comment:
We should add new message types at the end of the enum as it will allow some mismatch between service and application versions.